catholic-university-leuven-vancouver.csl

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" page-range-format="minimal" demote-non-dropping-particle="sort-only" default-locale="en-GB">
+  <info>
+    <title>Catholic University of Leuven - Vancouver</title>
+    <id>http://www.zotero.org/styles/catholic-university-leuven-vancouver</id>
+    <link href="http://www.zotero.org/styles/catholic-university-leuven-vancouver" rel="self"/>
+    <link href="http://www.zotero.org/styles/bmj" rel="template"/>
+    <link href="https://bib.kuleuven.be/training-en-tutorials/vancouver-2023" rel="documentation"/>
+    <link href="https://service.elsevier.com/app/answers/detail/a_id/22175/p/16075/supporthub/mendeley/" rel="documentation"/>
+    <author>
+      <name>Tibo Verplancke</name>
+      <uri>https://www.linkedin.com/in/tibo-verplancke/</uri>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="science"/>
+    <summary>Superscript Vancouver style adapted to the standards of the Catholic University at Louvain</summary>
+    <updated>2024-10-27T21:09:46+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always" delimiter-precedes-et-al="never"/>
+      <label form="short" prefix=" (" suffix=")"/>
+      <et-al font-style="normal"/>
```

Comment:
Remove this whole line. It doesn't do anything (anymore). You have several throughout the style.